canonix/tests/Main.hs

```diff
@@ -0,0 +1,29 @@
+import           Test.Hspec
+import           Paths_canonix (getDataFileName)
+import           Format (format)
+import           System.Directory (listDirectory)
+import           Control.Monad (forM_)
+import           Data.List (isSuffixOf)
+import qualified Data.ByteString as BS
+import qualified Data.ByteString.Lazy as BSL
```

Comment:
Of course `BL` is a better color for the bikeshed, because you get B Strict and B Lazy.
I have to be more strict about my laziness and not write this comment.
Oh too late :D